src/plugins/auto_batch/src/compiled_model.cpp

```diff
@@ -59,8 +59,9 @@ CompiledModel::GetWorkerInferRequest() const {
     if (!batch_id) {  // need new request
         m_worker_requests.push_back(std::make_shared<WorkerInferRequest>());
         auto workerRequestPtr = m_worker_requests.back().get();
-        workerRequestPtr->_infer_request_batched = {m_compiled_model_with_batch->create_infer_request(),
-                                                    m_compiled_model_with_batch._so};
+        workerRequestPtr->_infer_request_batched._ptr = m_compiled_model_with_batch->create_infer_request();
+        if (workerRequestPtr->_infer_request_batched._so == nullptr)
```

Comment:
won't it always be nullptr? seem no one assign value to it

Reply:
can saw the discussion:
https://github.com/openvinotoolkit/openvino/pull/18172#discussion_r1267531918